.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ for more details
+#
+
+references:
+  working_directory: /src
+  defaults: &defaults
+    docker:
+      - image: golang:1.11-alpine
+
+  install_sw_requirements: &install_sw_requirements
+    run:
+      name: install_sw_requirements
+      command: |
+        apk add ca-certificates git openssh gcc libc-dev
+
+  get_dependencies: &get_dependencies
+    run:
+      name: get-dependencies
+      command: |
+        go get github.com/teamdigitale/anpr-dashboard-server/server
+        go get github.com/teamdigitale/anpr-dashboard-server/converter
+        go get github.com/stretchr/testify/assert
+
+  test: &test
+    run:
+      name: test
+      command: |
+        cd server && go test -race
```

Comment:
I found out that some other tests lived in other folder. In order to get them all I would avoid `cd server` and run test with all flag as `go test -race ./...`

Reply:
fixed with go 1.13